.release-please-manifest.json

```diff
@@ -1,12 +1,12 @@
 {
-  ".": "1.10.1",
-  "packages/api-explorer": "0.9.14",
+  ".": "1.11.0",
+  "packages/api-explorer": "0.9.15",
   "packages/code-editor": "0.1.7",
-  "packages/extension-api-explorer": "21.10.1",
+  "packages/extension-api-explorer": "21.11.0",
```

Comment:
oops, this should have been `21.10.2` with a `release-as` override in the `release-please-config.json` - oh well, not really important because we don't publish this to npm. again, with [this pending release-please pr](https://github.com/googleapis/release-please/pull/903) this will happen automatically in the future

Reply:
for now we'll just leave it as is till we bump to `21.12.0`

Reply:
Sorry I missed that.